Makefile

```diff
@@ -7,4 +10,9 @@ build_redist:
 	./build_redist.sh
 
 test-unit:
-	go test $(shell go list ./... | grep -v /vendor/ | grep -v /template/ | grep -v build) -cover
+	go test $(shell go list ./... | grep -v /vendor/ | grep -v /template/ | grep -v /build/ | grep -v /sample/) $(TEST_OPTS) -cover
+test-unit-docker:
+	docker run --rm \
```

Comment:
I'm not sure we need this?

Reply:
it's a shortcut to run test on unix-based image, since I'm on windows mostly